CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+<!-- omit in toc -->
+# Contributing to hardened-chromium
+
+First off, thanks for taking the time to contribute! ❤️
+
+> If you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
+> - Star the project
+> - Tweet about it
+> - Refer this project in your project's readme
+> - Mention the project at local meetups and tell your friends/colleagues
+
+<!-- omit in toc -->
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [I Have a Question](#i-have-a-question)
+- [I Want To Contribute](#i-want-to-contribute)
+  - [Building locally](#your-first-code-contribution)
+  - [Pull Requests](#💫-pull-requests)
+
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[Code of Conduct](https://github.com/secureblue/hardened-chromium/blob/master/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable behavior
+to <secureblueadmin@proton.me>.
+
+
+## I Have a Question
+
+> If you want to ask a question, we assume that you have read the available [Documentation]().
+
+Before you ask a question, it is best to search for existing [Issues](https://github.com/secureblue/hardened-chromium/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
+
+If you then still feel the need to ask a question and need clarification, we recommend the following:
+
+- Open an [Issue](https://github.com/secureblue/hardened-chromium/issues/new).
+- Provide as much context as you can about what you're running into.
+- Provide project and platform versions (nodejs, npm, etc), depending on what seems relevant.
+
+We will then take care of the issue as soon as possible.
+
+## I Want To Contribute
+
+> ### Legal Notice <!-- omit in toc -->
+> When contributing to this project, you must agree that you have authored 100% of the content, that you have the necessary rights to the content and that the content you contribute may be provided under the project license.
+
+### Building locally
+#### Setup
+Clone this repository:
+
+`git clone --depth 1 --recurse-submodules https://github.com/secureblue/hardened-chromium.git && cd hardened-chromium`
+
+Download chromium-%{version}-clean.tar.xz from the Fedora's server:
+
+`rpkg --path ./chromium sources`
+
+#### Harden
+Copy the patches inside the source folder:
+
+`cp vanadium_patches/* ./chromium && cp patches/* ./chromium`
```

Comment:
This looks great, thanks! 

The only nit I have is that the steps up to this point can be accomplished by running copr_script.sh I think?

Reply:
yes but copr_script copies all the files to the root of the project for copr to build. this is makes it unusable for local development. The steps are the same but everything happens inside the chromium folder.

Reply:
makes sense, thanks